src/app/core/translations/pl_PL.ts

```diff
@@ -35,6 +35,19 @@ export default {
   SELECT_LANGUAGE: 'Wybierz język',
   STATEMENT: 'Informator',
   ANNOUNCEMENTS: 'Ogłoszenia',
+  PLACEHOLDER_CHOOSE_DATE: 'Wybierz datę',
```

Comment:
Please move new translation keys at the end of the file so we don't mess  up the keys order with the "translations" file